meetings/2025-04-30.md

```diff
@@ -36,14 +40,54 @@
+* eP: of the main focuses of the CG should be data domains. They are more vertical domains, like ERP, but there are horizontal domains, like contacts and calendars. 
+* ... There are many details, events, contacts. There is a small number of shapes. We could try to prototype. There was a project during the mini-hackathon Jackson organized. There are use cases for indexing, pagination, etc.
+* HZ: It is complex subject, the problem is adoption. In relation of performance, Solid is not build for performance and it shuldn't be. People try to get benefits from Solid which they are never going to get. Even with databases, people came up with technologies for indexing. I don't think those should be part of Solid. I don't think we will be successfull if we try to have Solid deal with it. 
+* MH: I'm using Solid myself, I see two separate versions of Solid emerging. One is using Linked Data, another taking possesion of data objects and storing it in your store. I think you don't have control of data if you don't have custody of it. I see separation of one camp who links the data and another that stores that data.
+* HZ: I heard a shocking statement at IIW (internet identity workshop). Mostly people from EU and NA, some also from Asia. People were saying that we are too early with Solid. It was suggested that we need 3 or 4 things for self soverign systems. If you can't prove your identity, due to issuer of identity blocking you it's game over.
+* ...: The 2rd component is storage, if they cut access to your data you loose it. I need to prove my identity and own my data. The 3rd component is messaging, if I can't communicate it is again game over.
+* ...: There is also running algorithms. There is a big limitations of what I can do. The battles in the identity space is not over. There are sophisticated discussions in identity space. Whatever solution is adopted for identity, if storage and messaging doesn't support that, they will not be successful.
+* ...: In our work we need to be able to support those identity systems.
+* eP: I joined one of the EU groups, I have the impression that if we don't engage with this groups it will not help. We should engage more, crosspolinate and present reports on what happens in other groups.
+* ... Solid is more like generic servers, smart clients. The algorithms that can run on your data, that logic belongs to the client.
+* HZ: Personally I disaprove. From my experience, this doesn't scale. I had this converstation with Tim a couple of times. In my experience a lot of it would need to go into infrastructure. I tried to had this conversation in LWS. You will never have performance if you put a burden on the client, especially if it runs in the browser. 
+* eP: ...
+* HZ: For some reason we don't show examples of that.
```

Comment:
```suggestion
* HZ: Personally, I disapprove. From my experience, this doesn't scale. I had this conversation with Tim a couple of times. In my experience, a lot of it would need to go into infrastructure. I tried to have this conversation in LWS. You will never have performance if you put a burden on the client, especially if it runs in the browser. 
* eP: (???)
* HZ: For some reason we don't show examples of that.
```